src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { FilterOptions } from '../../types/FilterOptions';
+
+type Props = {
+  todos: Todo[];
+  selectedOption: FilterOptions;
+  selectOption: (option: FilterOptions) => void;
+};
+
+export const TodoFooter: React.FC<Props> = ({
+  todos,
+  selectedOption,
+  selectOption,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${todos.filter(todo => todo.completed === false).length} items left`}
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: selectedOption === -1,
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => {
+            selectOption(-1);
+          }}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
+            selected: selectedOption === false,
+          })}
+          data-cy="FilterLinkActive"
+          onClick={() => {
+            selectOption(false);
+          }}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={classNames('filter__link', {
+            selected: selectedOption === true,
+          })}
+          data-cy="FilterLinkCompleted"
+          onClick={() => {
+            selectOption(true);
+          }}
+        >
+          Completed
+        </a>
+      </nav>
```

Comment:
you cam map Object.values(Filter....)
and please use another emun values instead 1 -1